autosynth/providers/php.py

```diff
@@ -23,7 +23,7 @@ def list_repositories():
     # monorepo.
     repo = "googleapis/google-cloud-php"
 
-    tree = gh.get_tree(repo)
+    tree = gh.get_tree(repo, "master")
```

Comment:
This code will need to be revisited when they change their branch name to "main".

Can we save them the pain by calling `gh.get_default_branch()` here?

Reply:
Same story for the other languages too.